### ISSUE TITLE
$HOME or ~ as path

### ISSUE THREAD

**Issue by SRGOM** [dunst-project/dunst#350]
Please allow them as paths for running scripts

**Comment by CMCDragonkai** [comment]
Is this in version 1.3.2. When I use `~` in `icon_path`, dunst complains about not finding the icon for `dialog-information`.

**Comment by CMCDragonkai** [comment]
This is my current `icon_path`:

```
icon_path = ~/.nix-profile/share/icons/Adwaita/16x16/status/:~/.nix-profile/share/icons/Adwaita/16x16/devices/:~/.nix-profile/share/icons/hicolor/16x16/status/:~/.nix-profile/share/icons/hicolor/16x16/devices/
```

**Comment by tsipinakis** [comment]
@CMCDragonkai It's not, new release is scheduled for this weekend (and its very long overdue as you've noticed!)

**Comment by tsipinakis** [comment]
Oh, looking at this again, this actually _is_ in 1.3 but `icon_path` was skipped accidentally. You're looking for #520. It's been fixed in master and will be released with 1.4.
